### DISCUSSION TITLE
Admin password for K8s cluster multi-juicer ?

### DISCUSSION THREAD

**Discussion by KarlRanseier** [juice-shop/multi-juicer#261] [Q&A] [answered]
just created a K8s cluster on my local Docker following the guide:
https://github.com/juice-shop/multi-juicer/blob/main/guides/k8s/k8s.md 

When fetching the admin password as described:
_The password for the team gets auto generated if not specified, you can extract it from the kubernetes secret:
kubectl get secrets balancer-secret -o=jsonpath='{.data.adminPassword}' | base64 --decode_

i get back 'administrator' as password.

However entering this password on the Login-page does not let me in. (wrong password)

**Checking with DevTools looks, as if the Form cuts the password 'administrator' to 'administ'  ;-)**

**Comment by J12934** [comment]
Hi please note that the documentation is already adjusted for the upcoming v8 release.

The docs for the current stable version can be found under the last released tag.
https://github.com/juice-shop/multi-juicer/blob/v7.3.2/guides/k8s/k8s.md

Not sure why it would give you back administrator though. Its a 8 digit alpha numeric code that gets auto generated on every release unless you specify it with one of your one.

**Reply by KarlRanseier** [reply]
re-installed the cluster with the old guide (V 7.3.2) - but still the 
password seems to be 'administrator', which does not fit the login-form due to its length.

How is this part of the guide to be interpreted  - how can we not specify a password ?
_The password for the team gets auto generated **if not specified**, you can extract it from the kubernetes secret_

**Reply by KarlRanseier** [reply]
Changing the secret with _kubectl edit secret juice-balancer-secret_ (e.g. to 'administ') does also not help.

can it be, that the K8s secret is not updated to the correct value (against which the AuthN is performed)

**Reply by J12934** [reply]
sorry no idea whats going on.

Like I said before the admin passcode gets auto generated to a 8 char alpha numeric code unless you overwrite it yourself. (which the k8s guide your using doesn't do). the value from the secret is directly mounted into the balancer container. if you update the secret you'll have to restart the balancer deployment for it to take effect.

The word administrator doesn't even appear in the codebase at all, so whatever is setting the password to it very likely isn't something that multi juicer does. please double / quadrouble check your config

**Reply by KarlRanseier** [reply, marked as answer]
Deinstalled and reinstalled again - and **_before_** joining the admin-team on balancer, 
now the first time (for us) a generated secret for the admin team was present in the K8s secrets.

Then joining admin team we could log-on using this secret
